lib/config/presets/internal/replacements.ts

```diff
@@ -0,0 +1,19 @@
+import type { Preset } from '../types';
+
+export const presets: Record<string, Preset> = {
+  all: {
+    description: 'All replacements',
+    extends: ['replacements:jade-to-pug'],
+  },
+  'jade-to-pug': {
+    description: 'Jade was renamed to Pug',
+    packageRules: [
+      {
+        matchDatasources: ['npm'],
+        matchPackageNames: ['jade'],
+        replacementName: 'pug',
+        replacementVersion: '2.0.0',
```

Comment:
In the *vast* majority of cases you're going to need to add on to the PR that is created, for example to change references.

Reply:
> I saw in your example PR [^pr] that Jade was using a range, but the replacement PR pinned the version to `2.0.0`.
> 
> Is it intended that we "ignore" the existing range setup and instead pin the version?
> 
> [^pr]: https://github.com/JamieMagee/renovate-deprecation-replacements/pull/13

@JamieMagee I see you've left this earlier PR review comment open, did you make any changes before/while marking this PR "ready for review"?

Reply:
I left it open because I went back and forth on it, and to some degree I still am. Regardless, I should have provided an update.

For now I've made the required changes for replacements to respect the `rangeStrategy` option. We may want to default `rangeStrategy` to `pin` for replacements depending on user feedback.